src/main/webapp/i18n/de/result.json

```
@@ -84,6 +84,7 @@
             "preliminary": "vorläufig",
             "preliminaryTooltip": "Dein Ergebnis ist noch nicht endgültig, weil weitere Tests nach der Einreichungsfrist ausgeführt werden.",
             "preliminaryTooltipSemiAutomatic": "Dein Ergebnis ist noch nicht endgültig, weil weitere Tests nach der Einreichungsfrist ausgeführt werden oder eine manuelle Bewertung aussteht.",
+            "codeIssuesTooltip": "Die automatische Codeanalyse hat mögliche Probleme in deinem Code gefunden.",
```

```suggestion
            "codeIssuesTooltip": "Die automatische Codeanalyse hat mögliche Probleme in deinem Codestil gefunden.",
```
Or
```suggestion
            "codeIssuesTooltip": "Die automatische Codeanalyse hat Codeissues gefunden.",
```
"Probleme" might sound a bit to alarming to some students

---

Sure, ‘Codeissues’ sounds good.